nupic/research/frameworks/greedy_infomax/models/BilinearInfo.py

```diff
@@ -0,0 +1,198 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
```

Comment:
Files should be named using pothole case (e.g., `bilinear_info.py`) and not camel case (e.g., `BilinearInfo.py`). In contrast, python classes are named using camel case. Please change file names to pothole case where necessary:

- `BilinearInfo.py`
- `ClassificationModel.py`
- `FullModel.py`
- `ResNetEncoder.py`